### PR TITLE
[ADD] crm: email and manual leads

### PR DESCRIPTION
**Pull request by jero-odoo**
Adding a new doc to outline the process of generating CRM leads via email alias or manual creation. This is intended to replace the existing [Generating leads/opps doc.](https://github.com/documentation/17.0/applications/sales/crm/acquire_leads/generate_leads.html) (Which will be removed after both this doc and the new webform doc are published)

Docs task: https://www.odoo.com/mail/view?model=project.task&res_id=3931235&access_token=3a69927b-7d03-4f2f-a5b2-935fa74d1b05

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hey @brse-odoo this is ready for review. Thank you!

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+===================================
+
+.. |st-o| replace:: :icon:`fa-star-o`
+.. |st| replace:: :icon:`fa-star`
+
+*Leads* can be added to the *CRM* app from custom email aliases, and by manually creating new
+records. This is in addition to the leads and opportunities created in the app through the
+:doc:`website contact form <opportunities_form>`.
+
+First, ensure the leads feature is enabled on the database by navigating to :menuselection:`CRM app
```
Comment (brse-odoo) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:12]:
```suggestion
First, ensure the leads feature is enabled in the database by navigating to :menuselection:`CRM app
```
Optional - "in" sounded better than "on" but I could be wrong

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+To create or update a sales teams' email alias, navigate to :menuselection:`CRM app -->
+Configuration --> Sales Teams`. Click on a team from the list to open the team's details page.
+
+.. image:: email_manual/email-alias.png
+   :align: center
+   :alt: The sales team details page, focused on the email alias section.
+
+In the :guilabel:`Email Alias` field, enter a name for the email alias, or edit the existing name.
+In the :guilabel:`Accept Emails From` field, use the drop-down menu to alter who is allowed to send
```
Comment (brse-odoo) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:30]:
```suggestion
In the :guilabel:`Accept Emails From` field, use the drop-down menu to choose who is allowed to send
```

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+Manually create opportunities
+-----------------------------
+
+To manually create an opportunity, navigate to :menuselection:`CRM app --> Sales --> My Pipeline`.
+At the top-left of the page, click :guilabel:`New` to create a new opportunity Kanban card. In the
+:guilabel:`Organization/Contact` field, enter the name of the company the opportunity is for.
+
+Choose a name and enter it in the :guilabel:`Opportunity` field. *This is a required field.*
+
+.. tip::
```
Comment (brse-odoo) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:89]: An example could help this tip be a bit more useful - just a quick few more words

**Review by brse-odoo — APPROVED** [review]
Hi @jero-odoo I've finished my review of this PR, great job! Approving with very few suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

**Comment by jero-odoo** [comment]
Hey @ksc-odoo this is ready for final review. Thanks!

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+- |st| |st| |st-o|: high priority
+- |st| |st| |st|: very high priority
+
+.. note::
+   The information in the :guilabel:`Expected Revenue` and priority fields can be used to track
+   performance for individual salespeople, and on a team basis. See
+   :doc:`../performance/expected_revenue_report` and :doc:`../track_leads/lead_scoring` for more
+   information.
+
+Click :guilabel:`Add`.
```
Comment (ksc-odoo) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:116]:
```suggestion
Once all the necessary information has been entered, click :guilabel:`Add`.
```

**Review by ksc-odoo — APPROVED** [review]
hey, @jero-odoo -- just finished my Final Review. Awesome job! Approving now 👍 only a handful of minor suggestions that require your attention. Once you implement the necessary adjustments, feel free to tag this for Tech Review. Thanks! 🙏

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+- |st-o| |st-o| |st-o|: low priority
+- |st| |st-o| |st-o|: medium priority
+- |st| |st| |st-o|: high priority
+- |st| |st| |st|: very high priority
+
+.. note::
+   The information in the :guilabel:`Expected Revenue` and priority fields can be used to track
+   performance for individual salespeople, and on a team basis. See
+   :doc:`../performance/expected_revenue_report` and :doc:`../track_leads/lead_scoring` for more
+   information.
```
Comment (samueljlieber) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:115]: Suggestion to add a note about how the priority effects the order of leads in the Kanban view :)
```suggestion
.. note::
   Assigning a priority changes the order of leads in Kanban view, with higher priority leads
   displayed first.
```

File: content/applications/sales/crm/acquire_leads/email_manual.rst [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+   .. image:: email_manual/opportunity-example.png
+      :align: center
+      :alt: An example of an opportunity in the CRM pipeline.
+
+Enter the contact information for the opportunity in the :guilabel:`Email` and :guilabel:`Phone`
+fields.
+
+In the :guilabel:`Expected Revenue` field, enter an estimated value for the opportunity. Then, use
+the |st-o| :guilabel:`(star)` icons to assign a priority.
```
Comment (samueljlieber) [inline content/applications/sales/crm/acquire_leads/email_manual.rst:104]: Modify the order of instruction here to accommodate the note admonition I am suggesting :) 
```suggestion
In the :guilabel:`Expected Revenue` field, enter an estimated value for the opportunity.

.. note::
   The information in the :guilabel:`Expected Revenue` and priority fields can be used to track
   performance for individual salespeople, and on a team basis. See
   :doc:`../performance/expected_revenue_report` and :doc:`../track_leads/lead_scoring` for more
   information.

Then, use the |st-o| :guilabel:`(star)` icons to assign a priority.
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo! Awesome work on this new CRM doc– I really like how the information in this doc is structured 👏

Please update the commit message (using `git commit --amend`) to follow the naming convention:
```
[ADD] crm: email and manual leads
```
I am approving with a suggestion, thanks for your hard work on this!
.....
@robodoo delegate=jero-odoo

**Pull request closed without merging**